src/issue/screens/new-issue.screen.js

```diff
@@ -98,6 +99,39 @@ class NewIssue extends Component {
     }
   };
 
+  handleKeyboardStateChange = (state, event) => {
+    switch (state) {
+      case 'show':
+        Animated.parallel([
```

Comment:
I guess you have 2 animations at first. But now `Animated.parallel` is useless.

Reply:
Yeah. Sorry, I reused this code from my current project. `Animated.parallel` can be removed.